### DISCUSSION TITLE
Upgrade from 2.6.2 to latest

### DISCUSSION THREAD

**Discussion by austinbailey** [netbox-community/netbox#5615] [Q&A] [answered]
I have an instance of Netbox running version 2.6.2 and want to upgrade to the latest version. What is the best upgrade path? Can I go directly or do I need to upgrade to other versions first?

**Comment by ryanmerolle** [comment, marked as answer]
One of the better posts on this topic is by @candlerb 

https://github.com/netbox-community/netbox/discussions/6683#discussioncomment-943756
